mobile-app/.env

```diff
@@ -0,0 +1,2 @@
+EXPO_PUBLIC_CONTENTFUL_SPACE_ID=wp2l3qgiko1l
```

Comment:
Ne bismo na ovom public repo-u smeli da imamo konkretne secrete koje bi neko mogao kasnije da koristi i možda utiče na billing same platforme korišćenjem ovih kredencijala.

Ovaj `.env` fajl mora da ide u gitignore.

Dobra praksa je da napravimo `.env.example` gde se nalaze samo nazivi varijabli, bez konkretnih vrednosti, a da u readme dodamo kontakt organizacije kako bi neko dobio ove varijable. A na njhovom nalogu možemo napraviti jedan google dokument i šerovati sa programerima za sada. Znači bitno je da je organizacija owner ovih informacija.

Kako ti se to čini?